e2e/cypress.config.mjs

```diff
@@ -5,6 +5,9 @@ import installLogsPrinter from 'cypress-terminal-report/src/installLogsPrinter.j
 
 export default defineConfig({
   defaultCommandTimeout: 16000,
+  retries: {
+    runMode: 3,
+  },
```

Comment:
This treats all tests as flaky, right? I'm not sure that's the right move, as if a PR introduces a flakiness to a test, we would be less likely to notice, reducing the overall usefulness of the whole test suite.

Reply:
yes, may be better to pinpoint the particular flaky tests and look into retrying them